GUI/inventory_table.py

```diff
@@ -181,3 +182,51 @@ def update_inventory(current_user):
 
     add_medicine_button = ttk.Button(frame, text="Confirm", command = lambda:update_inventory(current_user))
     add_medicine_button.pack(pady=10)
+
+def validate_date_format(date_str):
```

Comment:
I tried some other dates too and it didn't stop me from entering the end date first, but also it didn't error, so I don't see any major issues with that.